feature-libs/user/profile/components/update-password/update-password-component.service.ts

```diff
@@ -93,7 +94,16 @@ export class UpdatePasswordComponentService {
     });
   }
 
-  protected onError(_error: Error): void {
+  protected onError(_error: HttpErrorModel | Error): void {
```

Comment:
I spoke with Patrick Labadie (@plabadie ), and he mentioned that this change could potentially lead to compilation errors in the customer's custom code.  so we might need to wait for a major version

Reply:
if customer call onError with type Error, it should pass the compilation. Why it make potential compilation error?